examples/60_search/README.txt

```diff
@@ -0,0 +1,7 @@
+.. _search_examples:
+
+===============
+Search Examples
+===============
+
+Examples on changing the search procedure of Auto-sklearn.
```

Comment:
this *on* shouldn't it be *of*, or even better just: Examples changing the...

Reply:
Yes, it should.